### PR TITLE
Use urlJoin and joinIpfsLocation utilities consistently

### PR DESCRIPTION
**Pull request by dapplion**

### CODE REVIEW

File: src/utils/appInstaller/index.ts [hunk 1]
```diff
@@ -167,7 +171,7 @@ async function _installExternalApp(
     const shortName = name.split('.')[0]
     const initialVersionArray = toApmVersionArray('1.0.0')
     const contentUriHttpFromPublicGateway = utf8ToHex(
-      `http:${ipfsGateway}${getContentHash(contentURI)}`
+      'http:' + joinIpfsLocation(ipfsGateway, getContentHash(contentURI))
```
Comment (dapplion) [inline src/utils/appInstaller/index.ts:174]: @0xGabi This was the actual line creating the bug, where the URL was not properly joined so the second vault instance was trying to be fetched from a broken url

File: src/utils/apm/utils.ts [hunk 2]
```diff
@@ -52,6 +53,21 @@ export function stipIpfsPrefix(ipfsDirtyHash: string): string {
...
+ * the "ipfs" route only once, stripping it from the gateway and the location
+ * @param ipfsGateway "https://ipfs.io"
+ * @param location "Qmzz"
+ * @return "https://ipfs.io/ipfs/Qmzz/artifact.json"
+ */
+export function joinIpfsLocation(
+  ipfsGateway: string,
+  location: string
+): string {
+  return urlJoin(stipIpfsPrefix(ipfsGateway), 'ipfs', stipIpfsPrefix(location))
```
Comment (sohkai) [inline src/utils/apm/utils.ts:68]: I think we misspelled `strip` here

File: src/utils/apm/utils.ts [hunk 3]
```diff
@@ -52,6 +53,21 @@ export function stipIpfsPrefix(ipfsDirtyHash: string): string {
   )
 }
 
+/**
+ * Returns a joined IPFS location given an IPFS gateway and an IPFS path
+ * This util makes sure the url is properly joined, and that it contains
+ * the "ipfs" route only once, stripping it from the gateway and the location
+ * @param ipfsGateway "https://ipfs.io"
+ * @param location "Qmzz"
+ * @return "https://ipfs.io/ipfs/Qmzz/artifact.json"
```
Comment (sohkai) [inline src/utils/apm/utils.ts:62]: Nitpick, wouldn't it only return `"https://ipfs.io/ipfs/Qmzz"`?

Comment (dapplion) [reply src/utils/apm/utils.ts:62]: Offloaded to https://github.com/aragon/buidler-aragon/issues/131

File: src/utils/apm/utils.ts (hunk 2 above)
Comment (dapplion) [reply src/utils/apm/utils.ts:68]: Offloaded to https://github.com/aragon/buidler-aragon/issues/131

**Pull request merged**